app/src/main/res/layout/activity_simple_map.xml

```diff
@@ -1,7 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
 <com.mapbox.maps.MapView xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
+    xmlns:mapbox="http://schemas.android.com/apk/res-auto"
     android:id="@+id/mapView"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    tools:context=".examples.SimpleMapActivity" />
\ No newline at end of file
+    mapbox:mapbox_cameraTargetLat="40"
+    mapbox:mapbox_cameraTargetLng="-74.5"
+    mapbox:mapbox_cameraZoom="9"
+    tools:context=".examples.SimpleMapActivity" />
```

Comment:
Guess we could remove those tools.

Reply:
I think this xml is also used in `SimpleMapActivity` and probably we could use a different xml for `AddOneMarkerSymbolActivity` if we want to add these properties?

Reply:
Yes, need to add a new one for AddOneMarkerSymbolActivity

Reply:
This layout file is used in many places, need to revert the changes on it.